TWT/static/css/index.css

```diff
@@ -149,11 +149,7 @@
     grid-gap: 1rem;
   }
 
-  .base-timer__svg{
-    display: none;
-  }
```

Comment:
the svg is not responsive, removing `display: none` will have other side effects